Practice3/Practice3.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
```

Comment:
could you please get rid of the unused dependencies?

Reply:
Done